libs/database/src/migrate-data.ts

```diff
@@ -0,0 +1,54 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+const migrateAddTableMailAithentication = async () => {
```

Comment:
A little writing leif.
```suggestion
const migrateAddTableMailAuthentication = async () => {
```

Reply:
😅 Thanks!